FWCore/Integration/test/test_finalpath.sh

```diff
@@ -0,0 +1,79 @@
+found thing 'beginRun' TEST
+found thing 'endRun' TEST
+EOF
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --schedule --task >& finalpath.log || die "failed test_finalpath_cfg.py --schedule --task" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_not_found.log - || die "differences for test_finalpath_cfg.py --schedule --task" $?
+
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --endpath >& finalpath.log || die "failed test_finalpath_cfg.py --endpath" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_empty.log - || die "differences for test_finalpath_cfg.py --endpath" $?
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --endpath >& finalpath.log || die "failed test_finalpath_cfg.py --schedule --endpath" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_empty.log - || die "differences for test_finalpath_cfg.py --schedule --endpath" $?
+
+
+cat <<EOF > finalpath_expected_found.log
+found thing '' TEST
+found thing '' TEST
+found thing '' TEST
+found thing 'beginLumi' TEST
+found thing 'endLumi' TEST
+found thing 'beginRun' TEST
+found thing 'endRun' TEST
+EOF
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --endpath --task >& finalpath.log || die "failed test_finalpath_cfg.py --endpath --task" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_found.log - || die "differences for test_finalpath_cfg.py --endpath --task" $?
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --endpath --task >& finalpath.log || die "failed test_finalpath_cfg.py --endpath --task --schedule" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_found.log - || die "differences for test_finalpath_cfg.py --endpath --task --schedule" $?
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --path --task >& finalpath.log || die "failed test_finalpath_cfg.py --path --task" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_found.log - || die "differences for test_finalpath_cfg.py --path --task" $?
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --path --task >& finalpath.log || die "failed test_finalpath_cfg.py --path --task --schedule" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_found.log - || die "differences for test_finalpath_cfg.py --path --task --schedule" $?
+
+
+cat <<EOF > finalpath_expected_filter.log
+did not find thing '' TEST
+found thing '' TEST
+did not find thing '' TEST
+found thing 'beginLumi' TEST
+found thing 'endLumi' TEST
+found thing 'beginRun' TEST
+found thing 'endRun' TEST
+EOF
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --path --task >& finalpath.log || die "failed test_finalpath_cfg.py --path --filter" $?
+grep "thing '.*' TEST" finalpath.log | diff finalpath_expected_found.log - || die "differences for test_finalpath_cfg.py --path --filter" $?
+
+
+cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --path --task >& finalpath.log || die "failed test_finalpath_cfg.py --path --filter --task" $?
```

Comment:
Should this have
```suggestion
cmsRun  ${LOCAL_TEST_DIR}/test_finalpath_cfg.py -- --path --filter --task >& finalpath.log || die "failed test_finalpath_cfg.py --path --filter --task" $?
```
?

Reply:
fixed